### DISCUSSION TITLE
پیام ناظر بازار

### DISCUSSION THREAD

**Discussion by engAmirEng** [Glyphack/pytse-client#246] [Q&A]
سلام و ممنون از پکیج عالیتون
پیام ناظر بازار رو چطور میتونم بگیرم؟

**Comment by Glyphack** [comment]
سلام تا جایی که یادم هست این وجود نداره ولی میتونید کد پکیج رو ببینید شبیه اون رو خودتون بنویسید. میتونم اگه مشکلی بود کمک کنم.

**Comment by bhx98** [comment]
سلام و عرض ادب، یه سوال داشتم من روی یه نماد که ممنوع-متوقف هست میخاستم خروجی عدد پایانی، دیروز که در سایت بورس هم نشان داده میشه بگیرم ولی نتونستم، دیتافریم خالی هست،state را هم نتونستم واکشی کنم روی این نماد
ممنون میشم راهنمایی کنید
